Config/tile.py

```diff
@@ -6,4 +6,74 @@ def __init__(self,N=0,E=0,S=0,W=0):
 		self.N = N
 		self.E = E
 		self.S = S
-		self.W = W
\ No newline at end of file
+		self.W = W
```

Comment:
Could it be possible to use space instead of tabs for indentation? (PEP8 W191)

Reply:
Corrected in commit c459d849aec6f0e6e979d0acd679c2dcc7114783